lib/sinon/util/fake_xml_http_request.js

```diff
@@ -492,6 +492,12 @@ extend(FakeXMLHttpRequest.prototype, sinonEvent.EventTarget, {
         }
     },
 
+    setStatus: function setStatus(status) {
+        verifyRequestOpened(this);
+        this.status = typeof status === "number" ? status : 200;
+        this.statusText = FakeXMLHttpRequest.statusCodes[this.status];
```

Comment:
Not a big deal, but you can shorten this a bit and just use `status` when looking up the `statusText`.

```javascript
this.statusText = FakeXMLHttpRequest.statusCodes[status];
```